jenkins/janitor.py

```diff
@@ -101,18 +101,20 @@ def clear_resources(project, col, resource):
             continue
 
         # construct the customized gcloud commend
-        base = ['gcloud', 'compute', '-q', resource.name]
+        base = ["gcloud", "compute", "-q", resource.name]
         if resource.status is not None:
             base.append(resource.status)
-        base.append('delete')
-        base.append('--project=%s' % project)
+        base.append("delete")
+        base.append("--project=%s" % project)
         if resource.condition:
-            base.append('--%s=%s' % (resource.condition, col))
+            base.append("--%s=%s" % (resource.condition, col))
 
         print "Try to kill %s - %s" % (col, list(items))
-        if subprocess.call(base + list(items)) != 0:
+        try:
+            subprocess.call(base + list(items))
```

Comment:
Did you mean to switch this to `subproess.check_call`?

It is not clear to me why you are making this change.